ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/converter/PrefixAdderImpl.java

```diff
@@ -64,6 +67,7 @@ private void addNameSpacesAsPrefix(Model model, Map<String, String> nameSpaceMap
 				model.setNsPrefix(prefix, uri);
 			} catch (PrefixMapping.IllegalPrefixException ignored) {
 				// If namespace cannot be added as prefix, ignore.
+				LOGGER.error("IlligalPrefixException ignored: " + ignored.getMessage());
```

Comment:
Since this doesn't break any functionality.. I would either DEBUG log this, or not log at all. 
@WLefever-Cegeka since i'm not 100% familiar with the code, what's your take on this?